### PR TITLE
feat: update code to have single ibm ip variable

### PR DESCRIPTION
**Pull request by jor2**
### Description

update code to have single ibm ip variable

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

**Comment by vburckhardt** [comment]
I think we'll need to maintain an internal wrapper using this module + adding the additional variables that have been removed as internal only.

**Comment by jor2** [comment]
> I think we'll need to maintain an internal wrapper using this module + adding the additional variables that have been removed as internal only.

https://github.ibm.com/GoldenEye/iam-account-settings-module/pull/469

File: variables.tf [hunk 1]
```diff
@@ -5,78 +5,11 @@
...
-variable "allowed_ip_schematics_uk_south" {
-  description = "Schematics UK South IP addresses/ranges in list"
-  type        = list(any)
-  default     = []
-}
-
-variable "allowed_ip_schematics_us" {
-  description = "Schematics US IP addresses/ranges in list"
+variable "ibm_approved_ip_addresses" {
+  description = "Base set of IBM approved IPs, which are documented and sourced from https://pages.github.ibm.com/ibmcloud/Security/guidance/network-access-guidelines.html#pre-approved-ips and https://cloud.ibm.com/docs/schematics?topic=schematics-allowed-ipaddresses"
```
Comment (ocofaigh) [inline variables.tf:12]: This is a public module so we should no reference anything about IBM IPs here...

Rename the variable: `allowed_ip_addresses`
With description: `Defines the IP addresses and subnets from which IAM tokens can be created for the account. Note: value should be a comma separated string.

**Review by ocofaigh — CHANGES_REQUESTED** [review]
see comment

**Review by ocofaigh — CHANGES_REQUESTED** [review]
please address failing pre-commit hooks

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**